HelpSource/scdoc.css

```diff
@@ -1,3 +1,8 @@
+:root {
+    --color-input-fg: var(--sc-text-color);
+    --color-input-bg: var(--sc-neutral-darker);
+ }
```

Comment:
Shouldn't these definitions be in theme files instead of here? I understand that it's to avoid repetitions, but it would also be nice to have all definitions in one place.